Sources/Swift/Tools/SentryLogger.swift

```diff
@@ -1,4 +1,5 @@
 @_implementationOnly import _SentryPrivate
+@_spi(Private) import Sentry._Hybrid
```

Comment:
This won't work on CocoaPods or SPM.

Reply:
Should also have added a comment in code though to make this clear. 🙇‍♂️